extension/lib/storeUserData.js

```diff
@@ -7,7 +7,7 @@ const { decorateError } = require('./shopgate/logDecorator')
 module.exports = async (context, { userData }) => {
   const user = ShopgateUser.create(context)
   try {
-    await user.store(userData)
+    await user.store({ userId: userData.id, userData: userData })
```

Comment:
there u have it, a comment. I believe this should work:
```suggestion
    await user.store({ userId: userData.id, userData })
```